packages/kbn-config-schema/src/references/reference.ts

```diff
@@ -22,7 +22,7 @@ import { internals, Reference as InternalReference } from '../internals';
 export class Reference<T> {
   public static isReference<V>(value: V | Reference<V> | undefined): value is Reference<V> {
     return (
-      value !== undefined &&
+      value != null &&
```

Comment:
__optional nit:__ I know we don't have any tests for `reference` at all, but would be great if you can add few, at least for this public static method.

Reply:
added